1650009424137	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1650009424419	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile 304
1650022577503	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1650022577503	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650022577507	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650022577562	Sync.SyncScheduler	DEBUG	Woke from sleep.
1650022577562	Sync.SyncScheduler	DEBUG	Online, will sync in 2s.
1650022577562	Sync.SyncScheduler	DEBUG	Next sync in 2000 ms. (why=wake_notification)
1650022577563	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650022579563	Sync.Service	DEBUG	User-Agent: LibreWolf/99.0 (Linux x86_64) FxSync/1.101.0.20220410100435.desktop
1650022579563	Sync.Service	INFO	Starting sync at 2022-04-15 11:36:19 in browser session Bk1wrVxKAFO-
1650022579563	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1650022579567	Sync.Status	INFO	Resetting Status.
1650022579567	Sync.Status	DEBUG	Status.service: success.status_ok => success.status_ok
1650022579569	FirefoxAccounts	DEBUG	FxAccountsProfileClient: Requested profile
1650022579569	FirefoxAccounts	DEBUG	getOAuthToken enter
1650022579569	FirefoxAccounts	TRACE	getCachedToken returning cached token
1650022579569	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1650022579570	Services.Common.RESTRequest	DEBUG	GET request to https://profile.accounts.firefox.com/v1/profile
1650022579571	Sync.SyncAuthManager	INFO	Getting sync key
1650022579572	Sync.SyncAuthManager	INFO	Getting a sync token from: https://token.services.mozilla.com/1.0/sync/1.5
1650022579572	Sync.SyncAuthManager	DEBUG	Getting a token using OAuth
1650022579572	FirefoxAccounts	DEBUG	getOAuthToken enter
1650022579572	FirefoxAccounts	TRACE	getCachedToken returning cached token
1650022579572	FirefoxAccounts	DEBUG	getOAuthToken returning a cached token
1650022579573	Services.Common.TokenServerClient	DEBUG	Beginning OAuth token exchange: https://token.services.mozilla.com/1.0/sync/1.5
1650022579573	Services.Common.RESTRequest	DEBUG	GET request to https://token.services.mozilla.com/1.0/sync/1.5
1650022579577	Services.Common.RESTRequest	DEBUG	GET https://profile.accounts.firefox.com/v1/profile failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1650022579578	FirefoxAccounts	ERROR	Background refresh of profile failed: {"name":"FxAccountsProfileClientError","code":null,"errno":998,"error":"NETWORK_ERROR","message":"[Exception... \"NS_ERROR_UNKNOWN_HOST\"  nsresult: \"0x804b001e (NS_ERROR_UNKNOWN_HOST)\"  location: \"JS frame :: resource://services-common/rest.js :: onStopRequest :: line 488\"  data: no]"}
1650022579578	Services.Common.RESTRequest	DEBUG	GET https://token.services.mozilla.com/1.0/sync/1.5 failed: 2152398878 - NS_ERROR_UNKNOWN_HOST
1650022579578	Sync.SyncAuthManager	ERROR	Non-authentication error in _fetchTokenForUser: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1650022579579	Sync.Status	DEBUG	Status.login: success.login => error.login.reason.network
1650022579579	Sync.Status	DEBUG	Status.service: success.status_ok => error.login.failed
1650022579579	Sync.SyncAuthManager	ERROR	Failed to fetch a token for authentication: TokenServerClientNetworkError({"error":{}})(resource://services-common/tokenserverclient.js:39:36) JS Stack trace: TokenServerClientNetworkError@tokenserverclient.js:62:16
_tokenServerExchangeRequest@tokenserverclient.js:241:13
1650022579580	FirefoxAccounts	TRACE	not checking freshness of profile as it remains recent
1650022585388	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1650022585388	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1650022585388	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1650022585389	Sync.Service	DEBUG	User-Agent: LibreWolf/99.0 (Linux x86_64) FxSync/1.101.0.20220410100435.desktop
1650022585389	Sync.Service	INFO	Starting sync at 2022-04-15 11:36:25 in browser session Bk1wrVxKAFO-
1650022585389	Sync.Service	DEBUG	Exception calling anonymous function: LockException: Could not acquire lock. Label: "service.js: sync".(resource://services-sync/util.js:62:5) JS Stack trace: LockException@util.js:62:5
throwLockException@util.js:148:11
WrappedLock@util.js:162:15
_lockedSync@service.js:1370:6
sync/<@service.js:1334:18
WrappedCatch@util.js:133:27
sync@service.js:1335:7
1650022585390	Sync.Service	INFO	Cannot start sync: already syncing?
1650022585519	Sync.Resource	WARN	GET request to https://sync-1-us-west1-g.sync.services.mozilla.com/1.5/194602547/info/collections?v=1.101.0 failed: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1650022585520	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
1650022585520	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1650022585521	Sync.Telemetry	INFO	Early submission of sync telemetry due to changed IDs/NodeType
1650022585527	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "Abort"  nsresult: "0x80004004 (NS_ERROR_ABORT)"  location: "<unknown>"  data: no] No traceback available
